packages/core/src/sass/account-wizard.scss

```diff
@@ -21,10 +21,11 @@
         .account-wizard--error {
             display: flex;
             align-items: center;
-            justify-content: center;
+            justify-content: start;
             flex-direction: column;
             height: 100%;
             width: 100%;
+            padding-top: 13%;
```

Comment:
as discussed with Deveraj, the design already aligned the content to top with a little gap.

Reply:
@shayan-binary-2 , let's not use percentage values for padding please. The padding value should be fixed in `rem` or `px`.

Reply:
thanks Zaki, will keep it in mind